bedrock/base/templates/macros-protocol.html

```diff
@@ -197,7 +197,7 @@
 
 
 {# Picto Card: https://protocol.mozilla.org/patterns/molecules/picto-card.html #}
-{% macro picto_card(title=None, desc=None, class=None, heading_level=2, base_el='li', custom_desc=False) -%}
+{% macro picto_card(title=None, desc=None, class=None, heading_level=2, base_el='div', custom_desc=False) -%}
```

Comment:
Changing this does fix the issue on the /firefox/channel/desktop/ page, however in the majority of other pages this now creates invalid markup, because there the macro is commonly used as a list item, and a `<div>` shouldn't be a direct child of a `<ul>`.

So, to fix this bug fully you could either:

- Update every other instance of the macro  use `li` only where needed, and leave the default here set to `div`.
- Or alternatively set the macro on the /firefox/channel/desktop/ page to use `base_el='div'`, and leave the default value as `li`.

The first solution is probably the right thing to do, but it does require more work and testing than the second solution. Seeing as this is your own free time, I'll leave it up to you which option you want to go for here.